docs/DEVELOP.adoc

```diff
@@ -63,6 +63,7 @@ which acronym is not that important but here is a list of recommended ones (see
 * CHG: for a change requesting consideration when upgrading
 * DOC: for documentation aspects
 * WEB: anything regarding the website
+* DEV: relevant to developers
```

Comment:
No, I want to remove it, I was getting sick (as main developer) to have to think if it's relevant to the changelog or not. Developers have directly a look at the `git log` and basta, the changelog is for users. We can have the discussion in #825 and align the docs there.

Reply:
ok, removed